content/governance.md

```diff
@@ -0,0 +1,175 @@
+---
+title: Governance
+layout: page
+
+---
+
+# Governance
+
+This document describes the rules and governance of the project. It is meant to be followed by all the developers of the project and the Prometheus community. Common terminology used in this governance document are listed below:
+
+* **Team members**: Any members of the private [prometheus-team][team] Google group.
+
+* **Maintainers**: Maintainers lead an individual project or parts thereof ([MAINTAINERS.md][maintainers.md]).
+
+* **Projects**: Any repository in the Prometheus [GitHub organization][gh].
+
+* **The Prometheus Project**: The sum of all activities performed under this governance, concerning one or more Repositories or the community.
+
+## Values
+
+The Prometheus developers and community are expected to follow the values defined in the [CNCF charter][charter], including the [CNCF Code of Conduct][coc]. Furthermore, the Prometheus community strives for kindness, giving feedback effectively, and building a welcoming environment. The Prometheus developer community strives to operate by rough consensus for the majority of decisions and only resort to conflict resolution through a motion if consensus cannot be reached.
+
+## Projects
+
+Each project must have a MAINTAINERS.md file with at least one maintainer. Where a project has a release process, access and documentation should be such that more than one person can perform a release. Releases should be announced on the prometheus-users mailing list. Any new projects should be first proposed on the [developers mailing list][devs] following the voting procedures listed below. When a project is no longer relevant, it should be moved to the prometheus-junkyard Github organization.
+
+## Decision making
+
+### Team members 
+
+Team member status may be given to those who have made ongoing contributions for at least 3 months to the Prometheus Project. This is usually in the form of code improvements and/or notable work on documentation, but organizing events or user support could also be taken into account. 
```

Comment:
"...who have made ongoing contributions to the Prometheus project for at least 3 months." reads better to me (but I'm not a native speaker). Also, I don't think we should capitalize "Project" (or otherwise we should also capitalize Maintainers etc. if used as one of the terms defined in the preamble).

Reply:
@beorn7's version reads better to me.

Reply:
All the capitalizations of Project were in "The Prometheus Project". I changed all mentions of that, including the definition.

Reply:
All the capitalizations of Project were in "The Prometheus Project". I changed all mentions of that, including the definition.